src/test/java/com/EighthLight/app/TestGame.java

```diff
@@ -39,6 +39,8 @@ void startGameTiedWithCorrectInput() {
         Game game = new Game(ui, board, playerOne, playerTwo);
         game.start();
 
+        assertEquals(board, playerOne.getMakeMoveArgs().get(0));
```

Comment:
Nit pick, but I might move this below other assertions that come before it. Like `ui.display` gets called first in the loop so maybe this assertion should be below that?